### PR TITLE
Vis reg test suites for Text input component.

### PR DESCRIPTION
**Pull request by shaleen76**
JIRA: 

### LGTM's
- [x] Dev LGTM
- [x] Zoom LGTM

### CODE REVIEW

**Comment by shaleen76** [comment]
Made the required changes as per the review. 

multi-line-text
single-line-text

File: test/visreg/visreg.ts [hunk 1]
```diff
@@ -45,23 +45,30 @@ export function suite(browserName: string) {
...
         await driver.findElement(By.id('open-modal')).click();
-        await snappit.snap(`{browserName}/modals/open-modal`, $(util.selectors.modal));
+        await snappit.snap(`{browserName}/modals/open-modal`, $(util.selectors.components), { hide: [driver.findElement(By.css('.hxSiderail')) ] });
+    });
+
+    // Test Suite for Single Line Text for Chrome and Firefox browser
+    test("components/text-inputs", async t => {
+        await util.go(driver, "components/text-inputs");
+        await snappit.snap("{browserName}/text-inputs/single-line-text", $(util.selectors.singleLineText), { hide: [driver.findElement(By.css('.hxSiderail')) ] });
+        await snappit.snap("{browserName}/text-inputs/multi-line-text", $(util.selectors.multiLineText), { hide: [driver.findElement(By.css('.hxSiderail')) ] });
```
Comment (Droogans) [inline test/visreg/visreg.ts]: Hiding is no longer necessary here.

File: test/visreg/visreg.ts [hunk 2]
```diff
@@ -45,23 +45,30 @@ export function suite(browserName: string) {
...
     });
 
     test("components/open-modals", async t => {
         await util.go(driver, "components/modals");
         await driver.findElement(By.id('open-modal')).click();
-        await snappit.snap(`{browserName}/modals/open-modal`, $(util.selectors.modal));
+        await snappit.snap(`{browserName}/modals/open-modal`, $(util.selectors.components), { hide: [driver.findElement(By.css('.hxSiderail')) ] });
+    });
+
+    // Test Suite for Single Line Text for Chrome and Firefox browser
```
Comment (Droogans) [inline test/visreg/visreg.ts]: No need for this comment.

File: test/visreg/visreg.ts [hunk 3]
```diff
@@ -45,23 +45,30 @@ export function suite(browserName: string) {
         await snappit.snap("{browserName}/nav/custom-elements", $(util.selectors.nav));
     });
 
-    // Test Suite for Modal for Firefox browser
+    // Test Suite for Modal for Chrome and Firefox browser
```
Comment (Droogans) [inline test/visreg/visreg.ts]: No need for this comment.

File: test/common/util.ts [hunk 4]
```diff
@@ -51,7 +51,9 @@ export function $x(
  */
 export var selectors = {
     nav: "#stage > #nav",
-    modal: "#stage",
+    components: "#stage",
     tab: "//hx-tablist/hx-tab",
-    visreg: "*[data-visreg]"
+    visreg: "*[data-visreg]",
+    singleLineText: "div.demo",
+    multiLineText: "#multi-line-text-demo"
```
Comment (Droogans) [inline test/common/util.ts]: These are not a good candidates for a reusable selectors. Just inline these selectors into their respective `snap` calls directly.

```js
await snappit.snap("{browserName}/text-inputs/single-line-text", $("div.demo"));
```

and

```js
await snappit.snap("{browserName}/text-inputs/multi-line-text", $("#multi-line-text-demo"));
```

File: test/visreg/visreg.ts (hunk 3 above)
Comment (shaleen76) [reply test/visreg/visreg.ts]: Done

File: test/visreg/visreg.ts (hunk 2 above)
Comment (shaleen76) [reply test/visreg/visreg.ts]: Done

File: test/visreg/visreg.ts (hunk 1 above)
Comment (shaleen76) [reply test/visreg/visreg.ts]: Done

**Pull request merged**